src/utils.ts

```diff
@@ -223,6 +239,15 @@ export const setupEndorctl = async ({ version, checksum, api }: SetupProps) => {
     core.addPath(binPath);
 
     core.info(`Endorctl downloaded and added to the path`);
+
+    // Check to see if tsserver is installed -- if not install it (needed for javascript callgraphs)
+    const command = "tsserver";
+    core.info(`Checking for tsserver`);
+    if (!commandExists(command)) {
+      // Install it
+      core.info(`Installing tsserver`);
+      await exec.exec("npm", ["install", "-g", "typescript"]);
```

Comment:
Do you know if `npm` is globally available in the GitHub action? Or would this require an additional step in the workflow like `setup-node`?

Reply:
let me double check...

Reply:
@hghmn according to GPT

"Yes, specifying using: "node20" in your action.yml file means that Node.js 20 will be available to your GitHub Action, and npm will also be available by default. Since npm is bundled with Node.js, you’ll be able to use npm commands within your action without additional setup."

Since we specify our action uses node20 to run, npm should be available.

Reply:
Just tested an action where I installed node18 then used the endorlabs/github-action and it still worked.